arch/arc/core/cache.c

```diff
@@ -93,8 +76,9 @@ static void dcache_flush_mlines(u32_t start_addr, u32_t size)
 		return;
 	}
 
-	end_addr = start_addr + size - 1;
-	start_addr &= (u32_t)(~(DCACHE_LINE_SIZE - 1));
+	end_addr = (u32_t)start_addr + size - 1;
+	start_addr = (void *)(((u32_t)start_addr) &
+							 (u32_t)(~(DCACHE_LINE_SIZE - 1)));
```

Comment:
This is changing code, not renaming. :)

I mean, I get that it's only making no-op changes, but the point of my suggestion earlier was that reviewing changes like that gets harder the more unrelated changes are included.  And the meaning of the PR (in this case, renaming an ARC API so that it becomes part of a Zephyr generic cache framework) can get lost.

Basically: I think this should be squashed with the other patch.  In isolation it doesn't do anything.

Reply:
I shall close this PR then. Yes, I agree as those changes come together when the API names are also changed.  Thanks.